common/src/main/scala/com/gu/support/workers/helpers/Timing.scala

```diff
@@ -0,0 +1,27 @@
+package com.gu.support.workers.helpers
+
+import com.gu.support.workers.monitoring.CloudWatch
+import com.typesafe.scalalogging.LazyLogging
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+
+object Timing extends LazyLogging {
+
+  def record[T](cloudWatch: CloudWatch, metricName: String)(block: => Future[T]): Future[T] = {
```

Comment:
I've been talking to Roberto about potentially removing these Cloudwatch metrics from membership-frontend, since we pay for them but don't seem to use them.  I think we should leave them out of support-worker unless you think we will make use of them.